src/app/component/comments/components/comments-list/comments-list.component.scss

```diff
@@ -192,6 +187,16 @@
   }
 }
 
+@media screen and (max-width: 992px) {
+  .main-wrapper {
+    .cta-btn {
+      .btn-text {
+        font-size: 12px;
```

Comment:
super nesting)
angular has encapsulation for css, so try to avoid such nesting, because it is difficult to overwrite such selectors